deployments/fargate/README.md

```diff
@@ -0,0 +1,95 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html) for further reading.
+
+The
+[Splunk OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector)
+(splunk-collector or collector) can be deployed to an ECS Task as a standalone container 
+or as sidecar container alongside the application containers that you want to monitor.
+
+Find the Docker image repository for the collector
+[here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
```

Comment:
Suggested change: "See signalfx / splunk-otel-collector (https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags) for the Docker image repository for the collector."

Reply:
I edited the readme entirely. Please see later commit.